cmd/antrea-agent/agent.go

```diff
@@ -273,8 +287,10 @@ func run(o *Options) error {
 
 	// TODO: we should call this after installing flows for initial node routes
 	//  and initial NetworkPolicies so that no packets will be mishandled.
-	if err := agentInitializer.FlowRestoreComplete(); err != nil {
-		return err
+	if !enableBridge {
+		if err := agentInitializer.FlowRestoreComplete(); err != nil {
+			return err
+		}
```

Comment:
I'm not familiar with the whole workflow, but would it be really problematic to just move FlowRestoreComplete before the call to Initialize(), so we don't have different logic for enableBridge true or false?

Reply:
There is a reason why it's made so and called "FlowRestoreComplete", please check https://github.com/antrea-io/antrea/pull/658 for details. I think calling it earlier when AntreaIPAM will still have that problem